operator/Makefile

```diff
@@ -1,6 +1,9 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+APP_NAME = keda-manager
+IMG_REPO := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)
```

Comment:
Why we have doubled `$(DOCKER_PUSH_DIRECTORY)`? I can't imagine why.

Reply:
We build it in the same way in other projects that are using common `Makefile` (e.g [here](https://github.com/kyma-project/kyma/blob/main/components/function-controller/Makefile#L95)).

For output: `eu.gcr.io/kyma-project/keda-manager:PR-9`

```
DOCKER_PUSH_REPOSITORY=eu.gcr.io
DOCKER_PUSH_DIRECTORY=kyma-project
APP_NAME=keda-manager
DOCKER_TAG=PR-9
```

Reply:
My fault :(

I read `$(something)$(something)` (two identical strings) but there are two different `DOCKER_PUSH_REPOSITORY` and `DOCKER_PUSH_DIRECTORY`